### PR TITLE
Add instance methods to Animation and ChatPhoto

### PR DESCRIPTION
**Pull request by zeshuaro**

### CODE REVIEW

File: tests/test_animation.py [hunk 1]
```diff
@@ -72,20 +76,52 @@ def test_send_all_args(self, bot, chat_id, animation_file, animation, thumb_file
...
+        assert new_file.file_id == animation.file_id
+        assert new_file.file_path.startswith('https://')
+
+        new_file.download('game.gif')
+
+        assert os.path.isfile('game.gif')
+
+    @flaky(3, 1)
+    @pytest.mark.timeout(10)
+    def test_send_mp3_url_file(self, bot, chat_id, animation):
```
Comment (Bibo-Joshi) [inline tests/test_animation.py]: Please rename to `test_send_animation_url_file` ;)

Comment (zeshuaro) [reply tests/test_animation.py]: Fixed

**Comment by zeshuaro** [comment]
I've fixed the issues that you pointed out and added the new methods.

**Comment by Bibo-Joshi** [comment]
Yes, thanks!
The failed test seems unrelated, so this get's a LGTM from my side. :)

**Comment by tsnoam** [comment]
@zeshuaro Thank you for your contribution

**Pull request merged**